packages/codegen-ui-react/lib/forms/react-form-renderer.ts

```diff
@@ -414,6 +416,38 @@ export abstract class ReactFormTemplateRenderer extends StudioTemplateRenderer<
     statements.push(getInitialValues(formMetadata.fieldConfigs));
 
     statements.push(...getUseStateHooks(formMetadata.fieldConfigs));
+
+    // datastore relationship query
+    /**
+      const authorRecords = useDataStoreBinding({
+        type: 'collection',
+        model: Author,
+      }).items;
+    */
+    Object.values(formMetadata?.fieldConfigs).forEach((config) => {
+      if (config.relationship) {
```

Comment:
A couple comments here:
1. In line 442 of this file, there is already a place where we are iterating fieldConfigs. It'd be great to do this here so we're not doing it once more. 
2. Breaking this out as a helper in a new file in form-renderer-helper may help with maintainability
3. In line 494 of this file, the models are already added to imports, so no need to do it again :)

Reply:
One more thing - instead of getActionIdentifier, it'd be good to use getRecordsName from form-state.ts so that the variable name stays in sync

Reply:
Thanks, updated all of these.  A couple of these things were introduced recently so I didn't see them when writing this.

Reply:
Yea for sure. Thanks for updating